apps/api/src/domain/games/memory-match/models/memory-match-card.entity.ts

```diff
@@ -1,19 +1,58 @@
 import { LanguageCode, MultilingualTextItemRole } from '@coscrad/api-interfaces';
-import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import { UUID } from '@coscrad/data-types';
+import { isNonEmptyObject, isNullOrUndefined } from '@coscrad/validation-constraints';
+import { InternalError } from '../../../../lib/errors/InternalError';
 import { DeepPartial } from '../../../../types/DeepPartial';
 import { DTO } from '../../../../types/DTO';
 import { MultilingualText, MultilingualTextItem } from '../../../common/entities/multilingual-text';
 import { AggregateId } from '../../../types/AggregateId';
+import {
+    MissingAudioForMemoryMatchCardError,
+    MissingImageForMemoryMatchCardError,
+} from '../errors';
 
 export class MemoryMatchCard {
+    @UUID({
```

Comment:
This is a sequential ID, not a `uuid`. We should use `CountingNumber`, or at least `Integer`.

Reply:
But all that really matters is that this number is unique so `NonNegativeFiniteNumber` would suffice.